StructureDefinition/England-Observation.xml

```diff
@@ -0,0 +1,227 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="England-Observation" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/England-Observation" />
+  <version value="1.0.0" />
+  <name value="EnglandObservation" />
+  <title value="England Observation" />
+  <status value="draft" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="This profile allows exchange of information of Measurements and simple assertions made about an individual, device or other subject." />
```

Comment:
This description has changed from the previous version. It should be the description where it states it is constrained from UK Core and provides the Simplifier link.

Reply:
Done with the changes.